.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore .DS_Store files at the project level
```

Comment:
I manually deleted the file and added a project-local git ignore for now, let's track down what's going on with this on Friday!